wire-library/wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -1775,4 +1812,8 @@ static long valueToLong(@Nullable Object value) {
       return new BigInteger(string).longValue(); // Decimal.
     }
   }
+
+  private static boolean isStructNull(ProtoType type) {
```

Comment:
Will this be used in more places in the future? It's only used once right now so it could just be inlined.

Reply:
Inlined.